devfile.yaml

```diff
@@ -1,19 +1,48 @@
 apiVersion: 1.0.0
 metadata:
-  name: che-in-che
+  name: che-server-in-che
 projects:
   - name: che
     source:
       type: git
       location: 'https://github.com/eclipse/che.git'
 components:
-  - alias: che-dev
-    type: kubernetes
-    reference: .che-dev.yaml
+  - type: dockerimage
+    image: 'eclipse/che-dev:nightly'
+    alias: che-server-dev
+    mountSources: true
+    memoryLimit: 5Gi
```

Comment:
why we need 5GI here?

Reply:
I just didn't touch the value...

Reply:
3Gi should be enough

Reply:
> 3Gi should be enough

I'm not sure but it looks realistic after removing wsagent code from Che repo,

Reply:
I tested - it isn't. Maybe we should play with build command more.